### DISCUSSION TITLE
Receive And Fallback functions :  What if we directly pay ETH to a contract address without using receive or callback within the called contract?

### DISCUSSION THREAD

**Discussion by X0-100** [smartcontractkit/full-blockchain-solidity-course-py#1660] [Q&A] [answered]
```
    receive() external payable {
        fund()  ;
    }

    fallback() external payable {
        fund()  ;
    }
```

If we do not use receive or callback function in our contract and just go ahead and fund contract on our own.

**Comment by cromewar** [comment, marked as answer]
Hello @X0-100 
this is technically possible but not desirable, as the callback and receive functions has it's on internal events and secure measures to prevent attacks on your protocol, also is just a manner of good practice as the callback function will execute itself automatically anyway.

**Reply by X0-100** [reply]
Appreciate thanks!!
